CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 5.2.1
+
+## Changes from 5.2.1
+
+* Updates the `unityIAP` build flavor for compatibility with Unity IAP >= v3.1.0 for observer mode.
+
+## Changes from 5.2.0
```

Comment:
This is duplicated

Reply:
I don't think I understand. This is the same as the changelog of 5.2.0, which is already in lines 34-60

Reply:
yeah, good point. My thought was more for maybe the actual github release, it doesn't really make sense to have it here too

Reply:
oh I see. In any case, I would stick in the releases page to mention just the actual changes of each release. It might seem like 5.2.1 overshadows 5.2.0 at this moment because they were both released within the last hours, which will maybe make someone to miss 5.2.0, but when coming back to the releases page in the future, that effect will disappear and they will just look like two different releases. Do I make sense lol

Reply:
you do make sense, and I agree! updated